main/ahb_for_sles.go

```diff
@@ -421,13 +468,17 @@ func getExtensionAndRun() error {
 	initilizationInfo.UpdateCallback = updateCallbackFunc
 	vmExt, err := getVMExtensionFuncToCall(initilizationInfo)
 	if err != nil {
+		fmt.Fprintln(os.Stderr, "Extension initialization failed. Reason="+err.Error())
+		vmExt.ExtensionEvents.LogErrorEvent(
+			"Initialization",
+			fmt.Sprintf("AHBForSLES extension initialization finished. Result=Failure; Reason=%v", err.Error()))
```

Comment:
shouldn't we use `OPERATION_FAILURE_MSG` here ?
```go
fmt.Sprintf(OPERATION_FAILURE_MSG, "Initialization", err.Error())
```

Reply:
Fair point @jesusbv